crates/conformance-test-utils/src/madara/utils.rs

```diff
@@ -0,0 +1,450 @@
+use kakarot_rpc_core::client::constants::STARKNET_NATIVE_TOKEN;
```

Comment:
Just a note, when you are renaming files, it's a lot better to use `git mv src dst` to avoid these huge diffs.

Reply:
Ohh thanks I'll note this down 😄

Reply:
Can you also do it for this PR? It's easier for me to check the diff and see if you changed anything from the old `madara.rs`.